src/Assets/LightSkyDefense/Scripts/Credit.cs

```diff
@@ -7,14 +8,9 @@
 [RequireComponent(typeof(Interactable))]
 public class Credit : MonoBehaviour
 {
-    public int Value = 5;
-    private AudioSource source;
+    public int value = 5;
```

Comment:
Public properties should start with capital letter.

```suggestion
    public int Value = 5;
```